shiftcolumn/src/shiftcolumn.c

```diff
@@ -23,20 +23,12 @@
 # include "config.h"
 #endif
 
-#include "geany.h"
-#include "support.h"
+#include "geanyplugin.h"
 
 #ifdef HAVE_LOCALE_H
 # include <locale.h>
 #endif
 
-#include "ui_utils.h"
-
-#include "document.h"
-#include "keybindings.h"
-#include "plugindata.h"
-#include "geanyfunctions.h"
-
 #include <glib.h>
```

Comment:
This change fixes

  CC       shiftcolumn.lo
shiftcolumn.c: In function 'shift_left_cb':
shiftcolumn.c:95:4: warning: implicit declaration of function 'sci_has_selection' [-Wimplicit-function-declaration]
    if (sci_has_selection(sci)){
    ^
#include "sciwrappers.h" would also work but geanyplugin.h is recommended anyway

Reply:
This seems to have been fixed separately in f2ab51090839b2a93ec8cc3ed0e1e4874ff9b3cf.